config/gradle/publish.gradle

```diff
@@ -46,12 +33,10 @@ publishing {
                             deducedPublishRepo = "terasology"
                         }
 
-                        // Check the active Git branch and some module logic to see whether we're doing a release or snapshot
-                        String gitBranch = System.getenv()["BRANCH_NAME"]
-                        if (isMaster(gitBranch) && !shouldBypassModuleRelease()) {
-                            deducedPublishRepo += "-release-local"
-                        } else {
+                        if (project.version.toString().endsWith("-SNAPSHOT")) {
```

Comment:
I somehow like that this code is gone now 🙃 still being a bit afraid of what happens to module build chaos 🙈